contracts/modules/legacy/LegacyGateway.sol

```diff
@@ -3,191 +3,73 @@
 pragma solidity ^0.8.16;
 
 import "@openzeppelin/contracts-v4/token/ERC20/IERC20.sol";
-import "../../abstract/MasterAware.sol";
-import "../../interfaces/ILegacyClaims.sol";
-import "../../interfaces/ILegacyClaimsData.sol";
+import "../../abstract/MasterAwareV2.sol";
 import "../../interfaces/IERC20Detailed.sol";
 import "../../interfaces/IGateway.sol";
-import "../../interfaces/ILegacyIncidents.sol";
 import "../../interfaces/IMemberRoles.sol";
 import "../../interfaces/INXMToken.sol";
-import "../../interfaces/IPool.sol";
-import "../../interfaces/IQuotation.sol";
 import "../../interfaces/IQuotationData.sol";
-import "../../interfaces/ITokenController.sol";
 import "../../interfaces/ICover.sol";
-import "../cover/CoverMigrator.sol";
+import "../../interfaces/ICoverMigrator.sol";
 
-contract LegacyGateway is IGateway, MasterAware {
+contract LegacyGateway is IGateway, MasterAwareV2 {
 
   /* ============ CONSTANTS ============== */
 
   address public constant ETH = 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE;
 
   /* ========== STATE VARIABLES ========== */
 
-  IQuotation public quotation;
-  INXMToken public nxmToken;
-  ITokenController public tokenController;
-  IQuotationData public quotationData;
-  ILegacyClaimsData public claimsData;
-  ILegacyClaims public claims;
-  IPool public pool;
-  IMemberRoles public memberRoles;
+  // Removed the `quotation` state variable as we replaced MasterAware with MasterAwareV2,
+  // which occupies 2 slots instead of 1.
+  // address public _unused_quotation;
+
+  address public _unused_nxmToken;
```

Comment:
we've usually been naming them with numbers

but i don't see any issue with this, it's actually self-documenting, seems better